app/src/views/SignUpView.jsx

```diff
@@ -46,7 +47,11 @@ export class SignUpView extends React.Component {
       this.state.inputPassword
     );
   };
+
   render() {
+    if (this.props.isLoggedIn) {
```

Comment:
probably fine in this app, but in a full production app it's probably better UX to offer to log them out if they click on sign up